applications/convection_diffusion/rotating_hill/tests/convective_implicit_time_int.output

```diff
@@ -111,175 +111,175 @@ Setup Multistep time integrator ...
 Calculation of time step size according to CFL condition:
 
   CFL:                                       2.5000e-01
-  Time step size (CFL global):               1.0048e-03
+  Time step size (CFL global):               6.8342e-04
 
 Adjust time step size to hit end time:
 
-  Time step size:                            1.0040e-03
+  Time step size:                            6.8306e-04
 
 ... done!
 
 Starting time loop ...
 
 Calculate error for all fields at time t = 0.0000e+00:
-  Relative error (L2-norm): 5.63691e-03
```

Comment:
Why do we have these changes in errors? Almost an order of magnitude larger?

Reply:
Because now we are using a `DeformedCubeManifold`:
https://github.com/exadg/exadg/pull/586/files/8e95faacb4023d638fca4e13f7c53971bb2573b9#diff-2b1dc07e6c57bc454d21f974e7dcb5d08b9d1e66f0c0b04207e7ad4d38c9f7edR211-R215
by default in that application. It distorts the grid quite a bit, but I didi this to show that the mapping is not lost during AMR.

Reply:
I think we should not change the current test case. Another test case specifically for AMR is introduced here anyway, right?